plugin/src/test/java/io/jenkins/plugins/casc/SchemaGeneratorTest.java

```diff
@@ -0,0 +1,19 @@
+package io.jenkins.plugins.casc;
+
+import org.apache.commons.httpclient.HttpStatus;
+import org.junit.Test;
+import org.kohsuke.stapler.RequestImpl;
+import org.kohsuke.stapler.ResponseImpl;
+
+import java.io.IOException;
+
+public class SchemaGeneratorTest {
+
+    @Test
+    public void schemaShouldSucceed() throws IOException {
+
+        SchemaGenerator schemaGenerator = new SchemaGenerator();
+        System.out.println(schemaGenerator.generateSchema());
```

Comment:
@timja @casz This returns something like

`{"$schema":"http://json-schema.org/draft-04/schema#",
"title":"Schema Objects"
,"type":"object",
"additionalProperties":false,
"properties":{"foo":{"type":"string"}},
"required":["foo"]
}`